UPGRADE-2.0.md

```diff
@@ -325,6 +325,44 @@
         )
     ```
 
+## Grids
+
+The experimental `entities` filter has been removed. It has been replaced by the generic `entity` one.
+
+Before:
+```yaml
+sylius_grid:
+    grids:
+        # ...
+        sylius_admin_catalog_promotion:
+            # ...
+            filters:
+                channel:
+                    type: entities
+                    label: sylius.ui.channel
+                    form_options:
+                        class: "%sylius.model.channel.class%"
+                    options:
+                        field: product.channels.id
+```
+
+After:
+```yaml
+sylius_grid:
+    grids:
+        # ...
+        sylius_admin_catalog_promotion:
+            # ...
+            filters:
+                channel:
+                    type: entity
+                    label: sylius.ui.channel
+                    form_options:
+                        class: "%sylius.model.channel.class%"
+                    options:
+                        fields: [product.channels.id]
+```
```

Comment:
As only 2 lines (if I see correclty) have changed, maybe the diff would be more transparent, wdyt? 🤔 
```suggestion
```diff
sylius_grid:
    grids:
        # ...
        sylius_admin_catalog_promotion:
            # ...
            filters:
                channel:
-                   type: entities
+                   type: entity
                    label: sylius.ui.channel
                    form_options:
                        class: "%sylius.model.channel.class%"
                    options:
-                       field: product.channels.id
+                       fields: [product.channels.id]
```
```